packages/office-ui-fabric-react/src/components/Icon/FontIcon.tsx

```diff
@@ -11,21 +11,56 @@ export const getIconContent = memoizeFunction((iconName?: string) => {
     code: undefined
   };
 
+  const shouldReturnNull: boolean = code ? false : true;
+  if (shouldReturnNull) {
+    return null;
+  }
+
   return {
     children: code,
     iconClassName: subset.className,
     fontFamily: subset.fontFace && subset.fontFace.fontFamily
   };
 });
 
+export const getIconContentUnCached = memoizeFunction(
```

Comment:
Should you rather name it something like getIconContentWithValidIconCaching, since it does still cache the valid icons.

Reply:
Done